apis/workstations/v1alpha1/workstationcluster_types.go

```diff
@@ -67,22 +67,6 @@ type WorkstationClusterSpec struct {
 	PrivateClusterConfig *WorkstationCluster_PrivateClusterConfig `json:"privateClusterConfig,omitempty"`
 }
 
-type WorkstationClusterAnnotation struct {
```

Comment:
Not a blocker, but it looks like you're (only) trying to reuse the Annotation type - I don't think that is worth it generally.  Of course if you're also doing something else, not a problem.

Reply:
The "Annotation" type was not generated by our tooling. I had to create it manually. The original (generated) type for the field was `map[string]string`. Why not re-use the type?

Reply:
There's no harm in reusing the type, it might just be premature optimization (that can then require custom mappings).  In this case, sgtm!